exchanges/bybit/bybit.go

```diff
@@ -450,36 +405,49 @@ func (by *Bybit) GetBestBidAskPrice(ctx context.Context, symbol string) ([]Ticke
 		if err != nil {
 			return nil, err
 		}
-		tickers = append(tickers, TickerData{
-			resp.Data.Symbol,
-			resp.Data.BidPrice,
-			resp.Data.BidQuantity,
-			resp.Data.AskPrice,
-			resp.Data.AskQuantity,
-			resp.Data.Time.Time(),
-		})
-	} else {
-		resp := struct {
-			Data []bestTicker `json:"result"`
-			Error
-		}{}
+		return []TickerData{resp.Data}, nil
+	}
 
-		err := by.SendHTTPRequest(ctx, exchange.RestSpot, bybitBestBidAskPrice, publicSpotRate, &resp)
-		if err != nil {
-			return nil, err
-		}
-		for x := range resp.Data {
-			tickers = append(tickers, TickerData{
-				resp.Data[x].Symbol,
-				resp.Data[x].BidPrice,
-				resp.Data[x].BidQuantity,
-				resp.Data[x].AskPrice,
-				resp.Data[x].AskQuantity,
-				resp.Data[x].Time.Time(),
-			})
-		}
+	resp := struct {
+		Data []TickerData `json:"result"`
+		Error
+	}{}
+
+	err := by.SendHTTPRequest(ctx, exchange.RestSpot, bybitBestBidAskPrice, publicSpotRate, &resp)
+	if err != nil {
+		return nil, err
 	}
-	return tickers, nil
+
+	return resp.Data, nil
+}
+
+// GetTickersV5 returns tickers for either "spot", "option" or "inverse".
+// Specific symbol is optional.
+func (by *Bybit) GetTickersV5(ctx context.Context, category, symbol string) ([]Ticker, error) {
+	if category == "" {
+		return nil, errCategoryNotSet
+	}
+
+	val := url.Values{}
+	val.Set("category", category)
+
+	if symbol != "" {
+		val.Set("symbol", symbol)
+	}
+
+	result := struct {
+		Data struct {
+			List []Ticker `json:"list"`
+		} `json:"result"`
+		Error
+	}{}
+
+	err := by.SendHTTPRequest(ctx, exchange.RestSpot, bybitGetTickersV5+"/?"+val.Encode(), publicSpotRate, &result)
+	if err != nil {
+		return nil, err
+	}
+
+	return result.Data.List, nil
```

Comment:
Please don't filter the `Data` result here if there's more to the response, let callers of the API filter it. Yes its just "category"